ShearInX/ShearInX.h

```diff
@@ -0,0 +1,17 @@
+#ifndef XSHEAR_H
+#define XSHEAR_H
+
+// Must always be defined, but only used in test mode
+int errorFlag;
```

Comment:
if theyre only used in test mode, you could probably move them to the ShearInXTests.h